src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -48,11 +48,10 @@
 (defn- path-amount-out
   [path]
   (if (= (:bridge-name path) constants/bridge-name-hop)
-    (let [{:keys [token-fees bonder-fees amount-in]} path]
+    (let [{:keys [token-fees amount-in]} path]
       (-> amount-in
           money/from-hex
-          (money/sub token-fees)
-          (money/add bonder-fees)))
```

Comment:
@clauxx I am not sure about this change, but it didn't make sense to me that, for example, when trying to bridge 1 DAI, the receiving value was higher than the amount being sent. With this change, the value in the network card on the `To` column of the bridge screen is now the same as the estimated bridged value, which is what the designs show here -> https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=16655-240151&m=dev

This function is only used on `network-amounts-by-chain` utils function.

Reply:
it was done to match the calculations on hop's side (i tried their web dapp and compared the values on all chains). I also noticed the received value there sometimes being higher than the amount sent, I guess due to bonder fees being negative (maybe can floor it to 0 instead). The issue this fixed was that the received amount didn't account for the bonder fee, but I agree that we don't need different values for `To` and `Bridged to ...`. Maybe can also account for the bonder fees for the `To` value instead of removing it from `Received/Bridged to`?